pkg/tsdb/influxdb/fsql/fsql_test.go

```diff
@@ -21,9 +22,17 @@ type FSQLTestSuite struct {
 	suite.Suite
 	db     *sql.DB
 	server flight.Server
+	port   string
```

Comment:
maybe name this `addr` so it's clear what this is

```suggestion
	addr   string
```